flink-table-store-core/src/main/java/org/apache/flink/table/store/file/mergetree/compact/DeduplicateMergeFunction.java

```diff
@@ -48,8 +48,17 @@ public KeyValue getResult() {
         return latestKv;
     }
 
-    @Override
-    public MergeFunction<KeyValue> copy() {
-        return new DeduplicateMergeFunction();
+    public static MergeFunctionFactory<KeyValue> factory() {
+        return new Factory();
+    }
+
+    private static class Factory implements MergeFunctionFactory<KeyValue> {
+
+        private static final long serialVersionUID = 1L;
+
+        @Override
+        public MergeFunction<KeyValue> create(@Nullable int[][] projection) {
+            return new DeduplicateMergeFunction();
```

Comment:
Does this need to comment for unsupport of the nested projection?

Reply:
Actually, this class does not require projection relevant information.
In `PartialUpdateMergeFunction`, validation is not required too, because the `Project` class will throw an exception.